data/remnant missions.txt

```diff
@@ -1713,3 +1717,151 @@ mission "Remnant: Broken Jump Drive job"
+				goto main
+			label chilia
+			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "We meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly, "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
+			label main
+			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`
+			choice
+				`	"It's called a Void Rifle?"`
+				`	"I don't need details."`
+					goto end
+			`	"Yes, Void Rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacle's nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm not an engineer, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice it to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for clearing ships and tight quarters. Back then we had a number of threats that made building-to-building clearing operations an unfortunately frequent occurrence." His grim tone catches your attention, but he seems unwilling to pursue the topic.`
+			label end
+			`	"All right, enough history." He turns to the techs who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
```

Comment:
Again, `void rifle` is used here here but `Void Rifle` is used above. Since `void sprite` isn't capitalized, I'd do the same with `void rifle`.

Reply:
Change "Void Rifle" to "void rifle" everywhere except the name of the outfit